templates/ansible-drift.j2

```diff
@@ -28,6 +28,11 @@ cd "$dir" || exit 1
 if [ -n "$branch" ]; then
   git pull --quiet --ff-only $branch
   git submodule update --init --recursive --quiet
+
+  if [ -f requirements.yml ]; then
+    ansible-galaxy install -r requirements.yml --force
```

Comment:
I think we have to silence this, otherwise cron is gonna send a mail to root every night
```suggestion
    ansible-galaxy install -r requirements.yml --force > /dev/null
```